src/site/views/events/tmpl/future.php

```diff
@@ -15,7 +15,7 @@
 ?>
 
 	<!--</style>-->
-	<script type="text/javascript">
+	<script type="text/javascript" xmlns="http://www.w3.org/1999/html">
```

Comment:
xmlns Attribute is only generally used for html tags.
Also not really needed.
See http://www.w3schools.com/tags/att_html_xmlns.asp